Bellerophon/BellerophonTests/BellerophonTests.swift

```diff
@@ -11,6 +11,11 @@ import XCTest
 
 class BellerophonTests: XCTestCase {
 
+    lazy var mockManager = MockBPManager(config: BellerophonConfig(window: UIWindow(),
```

Comment:
Now that you provide a way to pass a force update view or not, can you add tests that cover the expected behavior when `forceUpdateView` is `nil` and when `forceUpdateView` is defined?

Reply:
will update